CHANGELOG.next.asciidoc

```diff
@@ -43,6 +43,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Journald: `include_matches.match` now behaves in the same way as matchers in `journalctl`. Users should carefully update their input configuration. {pull}40061[40061]
 - Journald: `seek` and `since` behaviour have been simplified, if there is a cursor (state) `seek` and `since` are ignored and the cursor is used. {pull}40061[40061]
 - Added `container.image.name` to `journald` Filebeat input's Docker-specific translated fields. {pull}40450[40450]
+- Added debug histograms for s3 object size and events per processed s3 object. {pull}40775[40775]
```

Comment:
We can move this changelog under CHANGELOG-developer.next.asciidoc since users won't care much about these debug metrics.

Reply:
Good suggestion. Done with https://github.com/elastic/beats/pull/40775/commits/1e91ecb8d8b8c564bb4c37c56741ae83c98113ec :)